@prisma/cli has been renamed to prisma

In https://www.howtographql.com/graphql-js/4-adding-a-database/, the instruction instructs installing prisma by 
```
npm install @prisma/cli --save-dev
```

However, it will return  the following as the module is renamed
```
npm ERR! code 1
npm ERR! path /Users/adriantam/src/hackernews-node/node_modules/@prisma/cli
npm ERR! command failed
npm ERR! command sh -c node scripts/preinstall-entry.js
npm ERR! ┌─────────────────────────────────────────────────────────────────────────────┐
npm ERR! │                                                                             │
npm ERR! │     The package @prisma/cli has been renamed to prisma.                     │
npm ERR! │                                                                             │
npm ERR! │     Please uninstall @prisma/cli first.                                     │
npm ERR! │     Then install prisma to continue using Prisma CLI:                       │
npm ERR! │                                                                             │
npm ERR! │         # Uninstall old CLI                                                 │
npm ERR! │         npm uninstall @prisma/cli                                           │
npm ERR! │                                                                             │
npm ERR! │         # Install new CLI                                                   │
npm ERR! │         npm install prisma --save-dev                                       │
npm ERR! │                                                                             │
npm ERR! │         # Invoke via npx                                                    │
npm ERR! │         npx prisma --help                                                   │
npm ERR! │                                                                             │
npm ERR! │     Learn more here: https://github.com/prisma/prisma/releases/tag/2.16.0   │
npm ERR! │                                                                             │
npm ERR! └─────────────────────────────────────────────────────────────────────────────┘

```

The corresponding require etc. needs to be updated as well.

--- a/content/backend/graphql-js/4-adding-a-database.md
+++ b/content/backend/graphql-js/4-adding-a-database.md
@@ -48,7 +48,7 @@ Speaking of being productive and building awesome stuff, let's jump back in and 
 First, let's install the Prisma CLI by running the following command in your terminal:
 
 ```bash(path=".../hackernews-node/")
-npm install @prisma/cli --save-dev
+npm install prisma --save-dev
 ```
 
 </Instruction>

